client/i18n/locales/english/translations.json

```diff
@@ -125,6 +125,7 @@
     "professional-certs-heading": "Earn free professional certifications:",
     "interview-prep-heading": "Prepare for the developer interview job search:",
     "upcoming-heading": "Upcoming curriculum:",
+    "legacy-certs-heading": "Legacy certifications:",
```

Comment:
```suggestion
    "legacy-curriculum-heading": "Explore our Legacy Curriculum:",
```

Reply:
The legacy JS cert is remaining claimable, though.

Reply:
hmm, but we're going to have the Legacy RWD, and Python for Everybody in there - those don't have certs.

Reply:
Do we not want to make a new Scientific Python cert? Or are the projects the same?

Reply:
the projects are the same